exercises/tournament/tests/tournament.rs

```diff
@@ -3,103 +3,103 @@ use tournament;
 #[test]
 fn just_the_header_if_no_input() {
     let input = "";
-    let expected = "Team                           | MP |  W |  D |  L |  P";
+    let expected = "Team                            | MP | W  | D  | L  | P";
 
     assert_eq!(tournament::tally(&input), expected);
 }
 
 #[test]
-#[ignore]
+// #[ignore]
 fn a_win_is_three_points_a_loss_is_zero_points() {
     let input = "Allegoric Alaskans;Blithering Badgers;win";
     let expected = "".to_string()
-        + "Team                           | MP |  W |  D |  L |  P\n"
-        + "Allegoric Alaskans             |  1 |  1 |  0 |  0 |  3\n"
-        + "Blithering Badgers             |  1 |  0 |  0 |  1 |  0";
+        + "Team                            | MP | W  | D  | L  | P\n"
+        + "Allegoric Alaskans              | 1  | 1  | 0  | 0  | 3\n"
```

Comment:
I will focus on one column.

Before:

```
before: |  1 |
after : | 1  | 
```

I think the previous (Before) version looks better, for the reason: imagine for a moment what happens if a two-digit number needs to be entered into this column. We'll disregard for a moment that a two-digit number never appears in any output in this exercise. If it were present, it would would have:

```
before: |  1 |
after : | 1  | 
2digit: | 10 | 
```

the `before` version seems to make more sense, since it is easier to scan the columns and understand that `10` is of a larger order of magnitude than `1`. The `after` version makes that harder to see.